protocol-specs/generator_outputs/lisk_codec/validArrayEncodings.json

```diff
@@ -0,0 +1,131 @@
+{
+	"title": "Valid array encodings",
+	"summary": "Examples of encoding arrays as required by lisk-codec",
+	"config": {
+		"network": "devnet"
+	},
+	"runner": "lisk_codec",
+	"handler": "validArrayEncodings",
+	"testCases": [
+		{
+			"description": "Encoding of array types",
+			"config": {
```

Comment:
not important, but why do you repeat the config here (and also on line 4)?

Reply:
here and in the other files as well

Reply:
I do agree with this but a schema for the output was defined and I have to respect it. So until we change the schema we need to keep it like this.